aspnetcore/security/authentication/microsoft-logins.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Microsoft Account external login setup | Microsoft Docs
+author: rick-anderson
+ms.author: riande
+manager: wpickett
+ms.date: 11/2/2016
+ms.topic: article
+ms.assetid: 66DB4B94-C78C-4005-BA03-3D982B87C268
+ms.technology: aspnet
+ms.prod: aspnet-core
+uid: security/authentication/microsoft-logins
+---
+# Configuring Microsoft Account authentication
+
+<a name=security-authentication-microsoft-logins></a>
+
+This tutorial shows you how to enable your users to log in with their Microsoft account using a sample ASP.NET Core project created in the [previous section](sociallogins.md).
+
+## Creating the app in Microsoft Developer Portal
+
+* Navigate to [https://apps.dev.microsoft.com](https://apps.dev.microsoft.com):
+
+![image](sociallogins/_static/MicrosoftDev.png)
+
+* Tap **sign in**:
+
+![image](sociallogins/_static/MicrosoftDevLogin.png)
+
+If you don't already have a Microsoft account, tap *[Create one!](https://signup.live.com/signup?wa=wsignin1.0&rpsnv=13&ct=1478151035&rver=6.7.6643.0&wp=SAPI_LONG&wreply=https%3a%2f%2fapps.dev.microsoft.com%2fLoginPostBack&id=293053&aadredir=1&contextid=D70D4F21246BAB50&bk=1478151036&uiflavor=web&uaid=f0c3de863a914c358b8dc01b1ff49e85&mkt=EN-US&lc=1033&lic=1)*. After signing in you will be redirected to *My applications* page:
+
+![image](sociallogins/_static/MicrosoftDevApps.png)
+
+* Tap **Add an app** in the upper right corner and enter your *application name*:
+
+![image](sociallogins/_static/MicrosoftDevAppCreate.png)
+
+* The *Registration* page will be displayed:
+
+![image](sociallogins/_static/MicrosoftDevAppReg.png)
+
+* Tap **Add Platform** in the *Platforms* section and select the **Web** platform:
+
+![image](sociallogins/_static/MicrosoftDevAppPlatform.png)
```

Comment:
Wrong aspect ratio. Screenshots taken on Microsoft Surface Book. Recommend changing Markdown to images with either width or height specified to benefit Surface and Retina users. Wontfix for now.